src/functions/durableOrchestration.ts

```diff
@@ -0,0 +1,36 @@
+import { app, HttpHandler, HttpRequest, HttpResponse, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { ActivityHandler, OrchestrationHandler } from 'durable-functions';
+
+// Replace with the name of your Durable Functions Activity
+const activityName = 'hello';
+
+const orchestrator: OrchestrationHandler = function* (context) {
+    const outputs = [];
+    outputs.push(yield context.df.callActivity(activityName, 'Tokyo'));
+    outputs.push(yield context.df.callActivity(activityName, 'Seattle'));
+    outputs.push(yield context.df.callActivity(activityName, 'Cairo'));
+
+    return outputs;
+};
+df.orchestration('durableOrchestrator1', orchestrator);
+
+const helloActivity: ActivityHandler<string> = (_context: InvocationContext, input: string) => {
+    return `Hello, ${input}`;
+};
+df.activity<string>(activityName, { handler: helloActivity });
+
+const clientInput = df.input.client();
+
+const clientHandler: HttpHandler = async (context: InvocationContext, request: HttpRequest) => {
+    const client = df.getClient(context, clientInput);
+    const instanceId = await client.startNew(request.params.orchestratorName, undefined, request.text());
```

Comment:
as a non-blocking concern: I don't love this `undefined` parameter. It would be great to find a way to not have to pass it if the user doesn't care about the generated `instanceId`.

Reply:
Yeah ideally I agree, but this is part of the client API completely separate from my new programming model changes. Since this would be a breaking change though, I wouldn't mind bundling it with the new programming model release